src/azul/deployment.py

```diff
@@ -159,7 +159,10 @@ def account(self):
 
     @property
     def account_name(self):
-        return one(self.iam.list_account_aliases()['AccountAliases'])
+        try:
```

Comment:
… we might as well do this logic in `azul`, too, thereby providing a generally useful config property without any caveats. 

Also: I forgot that we do have an easy way to tell if we are running inside Lambda or not: `config.lambda_context`. If the env variable is absent in Lambda, we should raise an error.

Reply:
I don't think we can use `config.lambda_context` as you suggest because it is set dynamically by the chalice middleware and we need to read the account name when the module is loaded (i.e. before any app methods are run) in order to create the S3 trigger.